src/common/action.h

```diff
@@ -20,6 +20,11 @@
 
 #include <glib.h>
 
+#define DT_READ_ACTION_ONLY -FLT_MAX
```

Comment:
I think there should be parenthesis around `#define` definitions.

Reply:
`move_size` indeeds need bracketing now that people are invited to feed something more complicated to `DT_PERFORM_ACTION`.